sdk/core/azure-core/azure/core/pipeline/transport/_aiohttp.py

```diff
@@ -139,7 +144,43 @@ def _get_request_data(self, request): #pylint: disable=no-self-use
             return form_data
         return request.data
 
-    async def send(self, request: HttpRequest, **config: Any) -> Optional["HTTPResponseType"]:
+    @overload
+    async def send(self, request: HttpRequest, **config: Any) -> Optional[AsyncHttpResponse]:
```

Comment:
Have you tried it?

Does it give more information in intellisense (curious)?

Reply:
yeah, basically instead of having one typehint, we get two. (see the 1/2 / 2/2 in the lower left of the typehint)

<img width="559" alt="Screen Shot 2021-09-28 at 3 51 09 PM" src="https://user-images.githubusercontent.com/43154838/135156223-07ab001a-72fb-4c8d-932c-8262a35ef518.png">


<img width="588" alt="Screen Shot 2021-09-28 at 3 51 14 PM" src="https://user-images.githubusercontent.com/43154838/135156290-52fac8f3-e77d-4256-bbbb-eef703c8352d.png">I also made sure to put the rest requests and responses one second, since it's still provisional